examples/src/main/scala/App.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+import com.microsoft.hyperspace._
+import com.microsoft.hyperspace.Hyperspace
+import com.microsoft.hyperspace.index.{IndexConfig, IndexConstants}
+
+object App {
+  def main(args: Array[String]): Unit = {
+    // Create Spark session
+    val sparkConf = new SparkConf().setMaster("local[*]")
+    val spark = SparkSession.builder.config(sparkConf).getOrCreate()
+    spark.sparkContext.setLogLevel("ERROR")
```

Comment:
Why error loglevel?

Reply:
I had added it to make the output less noisy with Info logs. Removed it.